### ISSUE TITLE
Panzer:  question about tPointLocationSearch.cpp test 

### ISSUE THREAD

**Issue by kddevin** [trilinos/Trilinos#3719]
The test panzer/adapters-stk/test/stk_interface_test/tPointLocationSearch.cpp is guarded with a test for ThreadPool being enabled.   As noted in comments in the relevant CMakeLists.txt, STKSearch had relied on ThreadPool, so this guard was needed.   However, STKSearch no longer depends on ThreadPool #3463, so this guard is no longer needed.

In #3445, we are removing ThreadPool from Trilinos.  For this test, I replaced the ThreadPool guard with guards for STKSearch and STKSearchUtil.  The test is then included in the builds, but does not compile for me.  It attempts to include file stk_search_util/stk_search_util/CreateBoundingBox.hpp, which does not exist.  With this include commented out, it cannot find type AxisAlignedBoundingBox3D.

I do not see this test built and run anywhere on the CDASH dashboard.  Should the test be removed?  If not, may I comment it out until a Panzer developer has time to look at it?

Thanks!

@trilinos/panzer 

@trilinos/<teamName>

## Your Environment

- **Relevant repo SHA1s:**  
- **Relevant configure flags or configure script:**  
- **Operating system and version:**  
- **Compiler and TPL versions:**  

## Related Issues

* Blocks 
* Is blocked by 
* Follows 
* Precedes 
* Related to 
* Part of 
* Composed of
